README.md

```diff
@@ -0,0 +1,20 @@
+# Babylon liquidity tool (BLT)
+## overview
+The Babylon liquidity tool is a tool to accept user liquidity, distribute it throughout the wider ecosystem. The vault reads allocation weights from the gauge. The gauge interface is designed in such a way that it supports a wide range of datasources to calculate the allocations. Potential sources of allocations range from
+- Shared security amount bought by BSNs
+- amount of economic btc security by BVSes
+- bbn voting
+- bribing pool
+- any combination of the above
+- anything expressable in uints
+
+The vault allocates liquidity to the different destinations through an adaptor interface. The adaptors are designed such that the vault works agnostic of the ecosystem it is allocating to. This allows the vault to support EVM, CosmWasm, MoveVM, SVM and bitVM. Each adaptor deployment supports one chain and one contract on that chain. For the allocation of the liquidity, a vault on the destination is needed.
```

Comment:
Using the adaptor pattern, an ecosystem agnostic design allows easy integration of different chains and protocols. would also move that down to the next section